deploy/autolabjs-stop.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#######################################################################
+# Purpose: To stop autolab components running
+# Author: Hrishikesh Dahiya
+# Date: 20-January-2019
+# Invocation: invoked as a cron job by root user; but can also be run as
+#             $sudo bash autolabjs-stop.sh
+########################################################################
+
+runningList=$(docker ps -q --filter "status=running" --format "{{.Names}}")
```

Comment:
This is going to collect names of all the containers in your system, not just the containers of AutolabJS. We need to play nice and only stop the containers started by AutolabJS.